### PR TITLE
Remove locationId in MakeNodeCommand

### PR DESCRIPTION
**Pull request by RMartinOscar**
Remove locationId as we don't use them anymore
Replace success1 & success2 with success and params
Set overallocate default as -1
Fix filament incorrect port range (1-65535 not 0-65536)
Enforce node_schema to only be either http or https

### CODE REVIEW

File: app/Console/Commands/Node/MakeNodeCommand.php [hunk 1]
```diff
@@ -70,6 +69,6 @@ public function handle(): void
         $data['daemon_base'] = $this->option('daemonBase') ?? $this->ask(__('commands.make_node.daemonBase'), '/var/lib/pelican/volumes');
 
         $node = $this->creationService->handle($data);
-        $this->line(__('commands.make_node.succes1') . $data['name'] . __('commands.make_node.succes2')  . $node->id . '.');
+        $this->line(__('commands.make_node.succes1') . $data['name'] . ' ' . __('commands.make_node.succes2')  . $node->id . '.');
```
Comment (Boy132) [inline app/Console/Commands/Node/MakeNodeCommand.php:72]: The node name and id should be variables in the translation.
Like `__('commands.make_node.success', ['name' => $data['name'], 'id' => $node->id])`
`Successfully created a new node with the name :name and has an id of :id`

**Review by Boy132 — CHANGES_REQUESTED** [review]

File: app/Console/Commands/Node/MakeNodeCommand.php (hunk 1 above)
Comment (lancepioch) [reply app/Console/Commands/Node/MakeNodeCommand.php:72]: oh goodness, can we also fix the "succes2" and "succes1"

Comment (Boy132) [reply app/Console/Commands/Node/MakeNodeCommand.php:72]: > oh goodness, can we also fix the "succes2" and "succes1"

That's what I meant :D

File: lang/en/commands.php [hunk 2]
```diff
@@ -39,8 +39,7 @@
         'daemonSFTP' => 'Enter the daemon SFTP listening port',
         'daemonSFTPAlias' => 'Enter the daemon SFTP alias (can be empty)',
         'daemonBase' => 'Enter the base folder',
-        'succes1' => 'Successfully created a new node with the name: ',
-        'succes2' => 'and has an id of: ',
+        'succes' => 'Successfully created a new node with the name :name and has an id of :id',
```
Comment (Boy132) [inline lang/en/commands.php:42]:
```suggestion
        'success' => 'Successfully created a new node with the name :name and has an id of :id',
```

File: app/Console/Commands/Node/MakeNodeCommand.php [hunk 3]
```diff
@@ -57,19 +57,19 @@ public function handle(): void
...
+        $data['cpu_overallocate'] = $this->option('overallocateCpu') ?? $this->ask(__('commands.make_node.cpu_overallocate'), '-1');
+        $data['upload_size'] = $this->option('uploadSize') ?? $this->ask(__('commands.make_node.upload_size'), '256');
         $data['daemon_listen'] = $this->option('daemonListeningPort') ?? $this->ask(__('commands.make_node.daemonListen'), '8080');
         $data['daemon_sftp'] = $this->option('daemonSFTPPort') ?? $this->ask(__('commands.make_node.daemonSFTP'), '2022');
         $data['daemon_sftp_alias'] = $this->option('daemonSFTPAlias') ?? $this->ask(__('commands.make_node.daemonSFTPAlias'), '');
         $data['daemon_base'] = $this->option('daemonBase') ?? $this->ask(__('commands.make_node.daemonBase'), '/var/lib/pelican/volumes');
 
         $node = $this->creationService->handle($data);
-        $this->line(__('commands.make_node.succes1') . $data['name'] . __('commands.make_node.succes2')  . $node->id . '.');
+        $this->line(__('commands.make_node.succes', ['name' => $data['name'], 'id' => $node->id]));
```
Comment (Boy132) [inline app/Console/Commands/Node/MakeNodeCommand.php:73]:
```suggestion
        $this->line(__('commands.make_node.success', ['name' => $data['name'], 'id' => $node->id]));
```

**Review by Boy132 — CHANGES_REQUESTED** [review]

**Review by Boy132 — APPROVED** [review]
LGTM

**Pull request merged**